jsonnet/obs-operator.jsonnet

```diff
@@ -1,9 +1,9 @@
 local t = (import 'github.com/thanos-io/kube-thanos/jsonnet/kube-thanos/thanos.libsonnet');
+local loki = import 'github.com/observatorium/deployments/components/loki.libsonnet';
 local config = import './operator-config.libsonnet';
-local obs = ((import 'vendor/github.com/observatorium/deployments/components/observatorium.libsonnet') + {
+local obs = ((import 'github.com/observatorium/deployments/components/observatorium.libsonnet') + {
```

Comment:
I am not sure if this way can be working in DS build. Maybe we need to have it vendorize as well.

Reply:
This is vendored. It is working just like before. The `jsonnet -J vendor/` flag tells jsonnet to import the files from the vendor folder and in there is a `github.com/observatorium/...` folder. Nothing really changed, except not using the symlinks anymore :)